crates/ty_python_semantic/resources/mdtest/import/partial_stub_packages.md

```diff
@@ -423,3 +423,48 @@ reveal_type(Both().both)  # revealed: str
 reveal_type(Impl().impl)  # revealed: Unknown
 reveal_type(Fake().fake)  # revealed: Unknown
 ```
+
+## Namespace stub with missing module
+
+Namespace stubs are always partial.
```

Comment:
Do you have a reference that backs this claim? This is somewhat surprising to me and I remember that I struggled to find good documentation for what's supposed to happen in some of those non obvious cases

Reply:
It would be great to add this link to the test